sdk/storage/Azure.Storage.Blobs.DataMovement/tests/BlobDataMovementTests.cs

```diff
@@ -11,14 +11,14 @@
 namespace Azure.Storage.Tests
 {
     /// <summary>
-    /// Base class for Blob tests
+    /// Base class for BlobDataMovement tests
     /// </summary>
     [ClientTestFixture(
     BlobClientOptions.ServiceVersion.V2020_06_12,
     BlobClientOptions.ServiceVersion.V2020_08_04,
     RecordingServiceVersion = BlobClientOptions.ServiceVersion.V2020_08_04,
     LiveServiceVersions = new object[] { BlobClientOptions.ServiceVersion.V2020_06_12 })]
-    public class BlobDataMovementTests : DataMovementTestBase
+    public class BlobDataMovementTests : BlobTestBase
```

Comment:
Changed the base so I could use non-static blob test helper methods, but maybe we'll have helpers in the DM base later that'll be needed here. How do we deal with needing both if that ends up being the case?

Reply:
Not sure what's the concern here. If you need both then it's fine? Or are you asking about overriding methods in the `BlobTestBase` ?

Reply:
Ah wait I see the comment in the .csproj file, give a me second.